types/uuid4/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for uuid4 2.0
+// Project: https://github.com/tracker1/uuid4
```

Comment:
```suggestion
// Project: https://github.com/tracker1/node-uuid4
```
is that correct project?

Reply:
Oh shoot yeah. That's the correct repo.